app/models/split_creation.rb

```diff
@@ -25,7 +25,11 @@ def weighting_registry=(registry)
   end
 
   def split
-    @split ||= app.splits.create_with(registry: merged_registry).find_or_initialize_by(name: name)
+    @split ||= app.splits.create_with(registry: merged_registry, feature_gate: feature_gate?).find_or_initialize_by(name: name)
+  end
+
+  def feature_gate?
+    name.end_with?("_enabled")
```

Comment:
Would we want something more explicit on the split to tag the type of split?

Reply:
ah, nvm, I see the flag below

Reply:
hm yeah, long term i would think probably something more explicit (although shorter term this gets the job done). we use this naming convention, but that's not to say others necessarily do. do we recommend it in documentation anywhere?

Reply:
I believe so but it might just be in a blog post or something.

Reply:
Yup, here's the [blog post](https://engineering.betterment.qa/2017/06/22/testtrack-implementation-best-practices.html#naming-split-tests-and-feature-flags) by the man himself @joejansen

Reply:
🆒 no external documentation though right? seems worth highlighting for other users that they'll get this functionality based on naming convention

Reply:
added to readme